doc/source/whatsnew/v1.5.0.rst

```diff
@@ -687,6 +687,7 @@ Plotting
 - Bug in :meth:`DataFrame.boxplot` that prevented specifying ``vert=False`` (:issue:`36918`)
 - Bug in :meth:`DataFrame.plot.scatter` that prevented specifying ``norm`` (:issue:`45809`)
 - The function :meth:`DataFrame.plot.scatter` now accepts ``color`` as an alias for ``c`` and ``size`` as an alias for ``s`` for consistency to other plotting functions (:issue:`44670`)
+- The function :meth:`DataFrame.plot.scatter` now passes errorbar keywords to ax.errorbar (:issue:`46952`)
```

Comment:
Can we make this more user-facing than mentioning `ax.errorbar` - perhaps that it now accepts `matplotlib` errorbar keywords to allow customizating the axis errorbar